gpu_ocean/demos/realisticSimulations/BasicSim.ipynb

```diff
@@ -8,7 +8,7 @@
     "This notebook sets up and runs a set of benchmarks to compare\n",
```

Comment:
Suggest deleting this file - it has no value any more

Reply:
I wish to keep it, as it is a visual reminder of the availability of the Norden domain :) I renamed it accordingly.